pyproject.toml

```diff
@@ -32,11 +32,20 @@ dependencies = [
     "prompt_toolkit",
     "numpy>=1.14.0",
     "pulp<2.8",
-    "ray>=2.30.0"
+    "ray>=2.30.0",
+    "h5py>=3.14.0",
+    "mne>=1.8.0",
+    "scikit-learn>=1.0.0",
```

Comment:
I don't think we _need_ to add these two at a package level. Any problems if we let these be strictly at the pipeline level? See my comment at the start of `utils/split.py`